Add Power Sankey graph documentation (#41819)
home-assistant/home-assistant.github.io: source/_dashboards/energy.markdown

layout: vertical
```

> ## Power Sankey graph

Reviewer: ```suggestion
## Power Sankey graph to visualize power flow
```

<p class='img'>

>   <img src='/images/dashboards/energy/sankey.png' alt='Screenshot of the power sankey graph card'>

Reviewer: ```suggestion
  <img src='/images/dashboards/energy/sankey.png' alt='Screenshot of the Sankey graph card to visualize power'>
```
Capitalize "Sankey" as it is a proper noun.

>   Screenshot of the power sankey graph card.

Reviewer: ```suggestion
  Screenshot of the power Sankey graph card.
```

</p>

> The power sankey graph shows the real-time flow of power in your home. Unlike the energy sankey card which shows historical energy data based on the selected date range, this card displays current power values and is not affected by the date picker selection.

Reviewer: ```suggestion
The power Sankey graph shows the real-time flow of power in your home. Unlike the energy Sankey card, which shows historical energy data based on the selected date range, this card displays current power values and is not affected by the date picker selection.
```

> It visualizes the instantaneous power flow from sources (like grid, solar panels, battery) to consumers in your home. Devices are grouped into floors and areas if these are configured.

Reviewer: ```suggestion
It visualizes the instantaneous power flow from sources (like the grid, solar panels, and battery) to consumers in your home. Devices are grouped into floors and areas if these are configured.
```

### YAML configuration

The following YAML options are available

{% configuration %}
type:
  required: true
  description: "`power-sankey`"
  type: string
collection_key:
  required: false
  description: "Collection key to use for the card. This links the card to a specific energy dashboard collection. Defaults to `energy_dashboard`."
  type: string
  default: energy_dashboard
title:
  required: false
  description: The title of the card.
  type: string
layout:
  required: false
  description: "`vertical`, `horizontal` or `auto`. Determines the orientation (flow direction) of the card. `auto` changes it based on the screen size."
  type: string
  default: auto
group_by_area:
  required: false
  description: Whether to group the devices by area
  type: boolean
  default: true
group_by_floor:
  required: false
  description: Whether to group the devices by floor
  type: boolean
  default: true
{% endconfiguration %}

### Examples

```yaml
type: power-sankey
```

The following example orients the flow from left to right:

```yaml
type: power-sankey
layout: horizontal
```

> ## Using Multiple Collections

Reviewer: ```suggestion
## Using multiple collections
```
